agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1833,11 +1833,9 @@ bool main_thread::hal_event_handler(bwl::base_wlan_hal::hal_event_ptr_t event_pt
                             !son::wireless_utils::is_low_subband(radio_info.channel)) {
                             LOG(DEBUG) << "iface " << iface
                                        << " is connected on low 5G band with filter, aborting";
-                            if (FSM_IS_IN_STATE(WIRELESS_ASSOCIATE_4ADDR_WAIT)) {
                                 FSM_MOVE_STATE(WIRELESS_CONFIG_4ADDR_MODE);
```

Comment:
Indentation is wrong in commit "agent: Fix bug which caused to move to the wrong backhaul state", fixed in a later commit. Same a few lines below. (nitpick)